data/menu.js

```diff
@@ -64,9 +64,8 @@ export const menuItems = [
       {
         id: 31,
         label: "Penetration Testing & Vulnerability Assessment",
-        link: "/portfolio-vapt",
```

Comment:
Why is this removed? The portfolio page is not opening now, it is directly sending the user to portfolio details page.

Reply:
Actually, dynamic route is being used to display the portfolio projects instead of static routes that's why the new route with title is used.
For the portfolio page the View Our Work (in hero section) and Work With Us (in the header) button are being used.
Also the same flow was followed for the "pages" nav link which also has a subMenu.

Let me know your opinion on this.

Reply:
As "Penetration Testing & Vulnerability Assessment" is a broad category, clicking it should open its page that lists different portfolio items related to it, it should not directly route us to the Mach7t (one of the portfolio item) page.

Reply:
Okay, got it. Additionally, there are more routes in the `app/(portfolio)` directory, such as `project-2`, `project-3`, etc. We need to fix those as well, but should I also add them to the `subMenu` of the `Portfolio` (NavLink)?

Reply:
No, leave them as is. They're dummy, I'll add more data though

Reply:
Okay, Thanks